cadctl/cmd/cluster-missing/cluster-missing.go

```diff
@@ -28,14 +29,24 @@ var ClusterMissingCmd = &cobra.Command{
 	Use:   "cluster-missing",
 	Short: "Will remediate the cluster-missing alert",
 	RunE: func(cmd *cobra.Command, args []string) error {
+		CAD_AWS_FOLDER, ok := os.LookupEnv("CAD_AWS_FOLDER")
+		if !ok {
+			return fmt.Errorf("could not load CAD_AWS_FOLDER envvar")
+		}
+
+		_, err := aws.NewClientFrmFileCredentials(CAD_AWS_FOLDER, "")
```

Comment:
discussed somewhere else, but this library API is less usable than others (gave you a replacement snippet for this)